documentation/pwa/tutorial-offline-form-submission.asciidoc

```diff
@@ -6,7 +6,7 @@ layout: page
 
 = Offline Form Submission
```

Comment:
It might be good to have a section like `image upload` section, that list some use cases and a cross-reference to the page documentation/client-side-forms/appendix-form-images.asciidoc

Reply:
Done.